tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -143,6 +160,52 @@ where
 		Ok(json!(json_value))
 	});
 
+	if let Some(state) = state {
+		// state_getStorage
+		let state_get_storage = "state_getStorage";
+		io.add_sync_method(state_get_storage, move |params: Params| {
+			// jsonrpc_core::serde::Deserialize<'de>` is not implemented for `H256`
+			// can not parse ShardIdentifier
+			// TODO: hanld error
+			match params.parse::<Vec<String>>() {
```

Comment:
because we send `vec![trusted_args.mrenclave.clone(), key]`, here use `[String; 2]` instread of `Vec<String>` can be better. so we can parse the key and shard_id conveniently.